src/events.rs

```diff
@@ -72,116 +76,118 @@ impl std::fmt::Debug for RawEvent {
     }
 }
 
-trait TypeSegmenter: Send {
+pub trait TypeSegmenter: DynClone + Send + Sync {
```

Comment:
I don't really understand why the `Sync` bound is required, can you explain?

Reply:
https://github.com/paritytech/substrate-subxt/pull/227/files#diff-b598fbef7e7017bd50ba2e1416836582326cec905b14d50bd5c6ac7904311aecL160